gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py

```diff
@@ -326,7 +359,10 @@ def test_DELETE_POOL_HEALTH_MONITOR_rpc_manager(self):
 
         """
 
-        self._test_rpc_manager('DELETE_POOL_HEALTH_MONITOR')
+        self._test_rpc_manager(
+            'DELETE_POOL_HEALTH_MONITOR',
+            self.fo.get_request_data_for_create_pool_hm(),
```

Comment:
Is the function call right? Getting request data for create pool HM? If this function is same for both create and delete, rename it to get_request_data_for_pool_hm()